tower-http/src/timeout/mod.rs

```diff
@@ -1,9 +1,11 @@
 //! Middleware for setting timeouts on requests and responses.
 
-mod body;
-mod service;
+pub mod body;
+pub mod service;
```

Comment:
I think I'd rather move the body docs to `TimeoutBody` and the service docs to this module. Rather than making the separate modules public.

Reply:
I have updated the PR with your preferences